src/models/UserInterestLower.js

```diff
@@ -0,0 +1,7 @@
+const UserInterestLower = 
+    `CREATE OR REPLACE TRIGGER UserInterestLower 
+    BEFORE INSERT OR UPDATE ON "UserInterest"
+    FOR EACH ROW
+    EXECUTE FUNCTION LowerInsert(interest)`;
+
+module.exports = UserInterestLower;
```

Comment:
Could this have been included in UserInterest.js as a named export? It's fine though.